src/templates/actor/pilot.hbs

```diff
@@ -4,23 +4,23 @@
     <div class="flexcol">
       <div class="flexrow">
         <div class="details">
-          <span>LICENSE LEVEL:</span>
+          <span>{{localize "lancer.pilot-sheet.header.ll"}}</span>
           <span class="pilot-callsign">{{mm.Level}}</span>
         </div>
 
         <div class="details flexcol">
-          <span>CALLSIGN:</span>
+          <span>{{localize "lancer.pilot-sheet.header.cs"}}</span>
           <span class="pilot-callsign">{{mm.Callsign}}</span>
         </div>
 
         <div class="details flexcol">
-          <span>NAME:</span>
+          <span>{{"pilot-sheet.header.name"}}</span>
```

Comment:
Interesting that this works... does seem to, but I'm curious how.

Reply:
First instinct is that it's a typo. Booting up the sheet, there are two empty spans resulting from it; one for name and one for background.

It would need to be `{{localize "pilot-sheet.header.name"}}`. I'd love if handlebars had a strict mode to help catch issues like this.

Reply:
First instinct is that it's a typo. Booting up the sheet, there are two empty spans resulting from it; one for name and one for background.

It would need to be `{{localize "lancer.pilot-sheet.header.name"}}`. I'd love if handlebars had a strict mode to help catch issues like this.

It should now work as intended.